### DISCUSSION TITLE
Thoughts on using keyword?

### DISCUSSION THREAD

**Discussion by jthoward64** [sequelize/sequelize#16237] [Help & Questions] [answered]
Javascript will be introducing a new keyword soon called using which will abstract away variable cleanup (sorta like cpp destructors, but c# has a direct equivalent). I feel like it could be interesting at least for transactions, but also probably for a few other concepts around Sequelzie.

**Comment by WikiRik** [comment, marked as answer]
Yes, it'll be a while before we can really take advantage of this but it looks interesting for sequelize
